XamarinCommunityToolkitSample/Models/WelcomeSectionId.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+
+namespace XamarinCommunityToolkitSample.Models
+{
+    public enum WelcomeSectionId
```

Comment:
I would still extract this to an Enums.cs file, not sure what others think of this though... @sthewissen  ?

Reply:
My opinion is that we should keep every class/struct/enum in the separate files.
But I'd like to make an exception for Enum extensions cuz that's the only option to add some methods there. (to comment below)

Reply:
Yeah I get it... still even in these cases I separated them. But like I said, not a big problem in this case, so if no other reviewer objects, current code can be kept.

Reply:
I'm fine with things being their own class.